app/screens/account_notifications/account_notifications.js

```diff
@@ -493,14 +516,52 @@ export default class AccountNotifications extends PureComponent {
                 />
             </Section>
         );
-    }
+    };
+
+    close = () => {
+        this.props.navigator.pop({animated: true});
+    };
 
     render() {
         const {theme} = this.props;
         const style = getStyleSheet(theme);
 
+        const navbarLeft = (
+            <BackButton
+                color={theme.sidebarHeaderTextColor}
+                onPress={() => preventDoubleTap(this.close, this)}
+                style={style.left}
+            />
+        );
+
+        const navbarTitle = (
+            <FormattedText
+                id='user.settings.modal.notifications'
+                defaultMessage='Notifications'
+                ellipsizeMode='tail'
+                numberOfLines={1}
+                style={[style.navTitle, {color: theme.sidebarHeaderTextColor}]}
+            />
+        );
+
+        const navbarRight = (
+            <ActionButton
+                actionEventName={SAVE_NOTIFY_PROPS}
+                enabled={true}
+                labelDefaultMessage='Save'
+                labelId='edit_post.save'
+                loadingEventName='saving_notify_props'
+            />
+        );
+
         return (
             <View style={style.wrapper}>
+                <StatusBar barStyle='light-content'/>
+                <NavBar
```

Comment:
The loading indicator should show in these, maybe we need to fix the styling for both IOS and Android, don't you agree? that unless we get rid of the load indicators on the right.

I'm ok with both options